code/hispania/modules/reagents/reagent_containers/food/cans.dm

```diff
@@ -0,0 +1,7 @@
+/obj/item/weapon/reagent_containers/food/drinks/cans/nuevabebida
+    name = "Coltrina"
+    desc = "Es una bebida con un gusto citrico pero dulce"
```

Comment:
in spanish needs to be translated. "a drink with a citric taste, but pretty sweet"